internal/docker/podman.go

```diff
@@ -0,0 +1,273 @@
+package docker
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/anchore/stereoscope/internal/log"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/homedir"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	// Podman supports the following connections
+	// rootless "unix://run/user/$UID/podman/podman.sock" (Default)
+	// rootfull "unix://run/podman/podman.sock (Default)
+	// remote rootless ssh://engineering.lab.company.com/run/user/$UID/podman/podman.sock
+	// remote rootfull ssh://root@10.10.1.136:22/run/podman/podman.sock
+
+	localRootlessPathTemplate = "/run/user/%d/podman/podman.sock"
+	defaultRemoteRootless     = "ssh://core@localhost:63753%s?secure=false"
```

Comment:
`secure=false` seems like a potential security problem (though unclear). Can you elaborate on what specifically is insecure about this connection?

Reply:
I copy-pasta'ed it from Podman's client code, and disabled it until I understand it. This flag is meant to control whether the ssh handshake validates the server's host key against the local known keys in `.ssh/known_hosts`, which is important when talking to remote Podman servers.